docs/guide/start/concept-quickstart.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Brooklyn Concepts Quickstart
+layout: website-normal
+menu_parent: index.md
+---
+
+The following section gives a quick summary of the main Brooklyn concepts you will encounter in Getting Started.  For later investigation of these concepts see [The Theory Behind Brookln]({{site.path.website}}/learnmore/theory.html), and the detailed description in [Brooklyn Concepts]({{site.path.guide}}/concepts/).
```

Comment:
My thought when working through the Getting Started was that it would be helpful to have an "in-line" definition of the main concepts, as part of Getting Started.  This is following our wider observation that the Getting Started could be more linear in form, and more self contained (reader doesn't have to jump to other pages to get information and then come back).  But if there's general impression that it would be preferable I can replace this with a link to the glossary?